Utilities/action_web.py

```diff
@@ -106,13 +106,34 @@ def get_element(self, page, element, platform_name, dict_save_value):
         for element_yaml in arr_element:
```

Comment:
please make  the changes to line 103
if dict_save_value:
instead of 
if dict_save_value is not None and dict_save_value.get(text) != None:

Reply:
Hi @qarampage, I have updated the code but I still keep checking the key in the dictionary